### DISCUSSION TITLE
How to reference a webp image in the public folder in the frontmatter of a post?

### DISCUSSION THREAD

**Discussion by acaladolopes** [arthelokyo/astrowind#648] [Q&A]
I have been trying to reference images in the public folder in the frontmatter like this:

image: /76e36ed0-913a-46ba-a04b-d10560d44d87.webp

but no matter what I try, I always get:

Failed to parse URL from /95cb42c2-118e-4d95-8659-f886b714ce72.webp at utils/remoteProbe.js:4:20

Thanks in advance

**Comment by ChrisHammond** [comment]
Is that image found in the dist folder after building? what's the path in dist?
